packages/stryker-karma-runner/src/starters/angularStarter.ts

```diff
@@ -1,21 +1,46 @@
+import * as decamelize from 'decamelize';
 import * as semver from 'semver';
 import { requireModule } from '../utils';
+import { NgConfigOptions, NgTestArguments } from '../StrykerKarmaSetup';
+import { getLogger, Logger } from 'stryker-api/logging';
+import * as path from 'path';
 
 const MIN_ANGULAR_CLI_VERSION = '6.1.0';
 
-export async function start(): Promise<void> {
-
+export async function start(ngConfig?: NgConfigOptions): Promise<void> {
+  const logger: Logger = getLogger(path.basename(__filename));
   // Make sure require angular cli from inside this function, that way it won't break if angular isn't installed and this file is required.
   const version = semver.coerce(requireModule('@angular/cli/package').version);
   if (!version || semver.lt(version, MIN_ANGULAR_CLI_VERSION)) {
-    throw new Error(`Your @angular/cli version (${version}) is not supported. Please install ${MIN_ANGULAR_CLI_VERSION} or higher`);
+    throw new Error(
+      `Your @angular/cli version (${version}) is not supported. Please install ${MIN_ANGULAR_CLI_VERSION} or higher`
+    );
   }
   let cli = requireModule('@angular/cli');
   if ('default' in cli) {
     cli = cli.default;
   }
+  const cliArgs = [
+    'test',
+    '--progress=false',
+    `--karma-config=${require.resolve('./stryker-karma.conf')}`
+  ];
+  if (ngConfig && ngConfig.testArguments) {
+    const testArguments: NgTestArguments = ngConfig.testArguments;
+    Object.keys(testArguments).forEach(key => {
+      const decamelizedKey = decamelize(key);
```

Comment:
I'm pretty sure that `decamlize('karmaConfig')` will result in `'karma_config'` instead of `'karma-config`. You can use `decamalize('karmaConfig', '-')` instead.

Reply:
done :-)